src/codegen/projection_translator.cpp

```diff
@@ -63,7 +66,7 @@ void ProjectionTranslator::PrepareProjection(
   }
 }
 
-void ProjectionTranslator::AddNonTrivialAttributes(
+/* static */ void ProjectionTranslator::AddNonTrivialAttributes(
```

Comment:
This is actually a comment, since C++ wouldn't allow us to write static in the function definition here.